internal/artifact/artifact.go

```diff
@@ -11,6 +12,27 @@ import (
 //go:generate stringer -type=Type
 type Type int
 
+func (t Type) String() string {
```

Comment:
this should not be needed, `go generate` should create this using `stringer`.

see line 12.

Reply:
Ah, now I see. This didn't work after a go get, of course. I'll remove it.